pa11ycrawler/html.py

```diff
@@ -2,19 +2,25 @@
 This script transforms JSON from the pa11ycrawler into a beautiful HTML
 report.
 """
-import os
-import os.path
-import io
 import re
 import argparse
-import shutil
 import json
 import logging
 import collections
-from mako.lookup import TemplateLookup
+from path import Path
+from jinja2 import Environment, PackageLoader
+from pa11ycrawler.util import pa11y_counts
 
 log = logging.getLogger(__name__)
 
+PARENT_DIR = Path(__file__).abspath().parent
+REPO_DIR = PARENT_DIR.parent
+
+# A WCAG ref consists of one or more uppercase letters followed by one or more
+# digits. The `code` that pa11y provides may have several WCAG refs -- if so,
+# they are separated by commas with no space in between.
```

Comment:
Your description is fine, but for simplicity, would you mind adding a couple of illustrative examples as well?